README.md

```diff
@@ -79,3 +72,32 @@ terraform init --upgrade
 ```console
 terraform apply
 ```
+
+## Help
+
+Have a feature request or found a bug? Let us know via [GitHub issues](https://github.com/iterative/terraform-provider-iterative/issues). Have questions? Join our [community on Discord](https://discord.gg/bzA6uY7); we'll be happy to help you get started!
+
+## Contributing
```

Comment:
I'd suggest to not spend too much space/text for contributors. Can we simplify this part or move it to docs.

Reply:
Maybe a separate `CONTRIBUTING.md` like in https://github.com/iterative/cml/pull/513?

Reply:
minified

Reply:
I'd prefer to keep somewhere a callout stating that `make install` and `terraform init --upgrade` are the development _abracadabra_. Probable near the `github.com/iterative/iterative` note.

Reply:
Else, it might be harder to figure out how to compile and test any modifications.

Reply:
6a0ffa2?